### PR TITLE
Logout button

### PR DESCRIPTION
**Pull request by akib-raihan-sazim**
**Description of changes**
- Logout button added on the home page to redirect to home page.

**Associated ticket link**
https://trello.com/c/94dpzN32

**Screenshot / Screen Recording**
Logout-Button

### CODE REVIEW

File: modules/HomePage/components/ToDoForm/ToDoForm.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```
Comment (ishmam-iqbal-sazim) [inline modules/HomePage/components/ToDoForm/ToDoForm.tsx:1]: Best to remove this.

File: pages/HomePage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import "@mantine/core/styles.css";
+import "@mantine/dates/styles.css";
+
+import HomePageContainer from "@/modules/HomePage/containers/HomePageContainer";
+
+const HomePage = () => {
```
Comment (ishmam-iqbal-sazim) [inline pages/HomePage/index.tsx:7]: Why is there a Home and a HomePage?

Comment (akib-raihan-sazim) [reply pages/HomePage/index.tsx:7]: I guess that came when I rebased it. I will remove that.

**Review by ishmam-iqbal-sazim — APPROVED** [review]

File: package-lock.json
Comment (ragib-hossain-sazim) [inline package-lock.json:1]: [JC] Why is `package-lock.json` getting updated here?

Comment (akib-raihan-sazim) [reply package-lock.json:1]: Not sure, this arise while I installed a package and rebased this brance with task-details branch

Comment (ragib-hossain-sazim) [reply package-lock.json:1]: If this change is not part of this PR please remove it.

**Review by ragib-hossain-sazim — APPROVED** [review]

**Pull request merged**